setup.md

```diff
@@ -4,166 +4,241 @@ This document will guide you through setting up Fennel on your
 computer. This document assumes you know how to download a Git
 repository and edit configuration files in a UNIX-like environment.
 
-## Requirements
+Fennel can be used in non-UNIX environments, but those environments
+will not be covered in this document.
+
+# Table of contents
```

Comment:
Can we skip the table of contents here and insert it using pandoc like we do with reference.md on fennel-lang.org? https://git.sr.ht/~technomancy/fennel-lang.org/commit/d1ac1f50d10f2fd09ac2d182c804e9b692a14374

IMO the table of contents is most useful when the hyperlinks work; when you're viewing it in your editor rather than your browser you don't have the benefit of being able to jump to a specific point.

Reply:
We sure can!